### ISSUE TITLE
Error cannot set logtostderr: no such flag -logtostderr

### ISSUE THREAD

**Issue by Constantin07** [kubernetes-sigs/dashboard-metrics-scraper#37]
Getting the below after upgrade to `v1.0.6`.
```
{"level":"error","msg":"Error cannot set logtostderr: no such flag -logtostderr","time":"2020-10-21T12:12:44Z"}
```
Is that supposed to be this way ?

PS: The deployment doesn't pass any explicit `args` or `command` statements.

**Comment by AshinWu** [comment]
I have the same problem in v.1.0.6 and how to fix it up?
![image](https://user-images.githubusercontent.com/27222274/109591681-eb459600-7b48-11eb-9170-5df518ad7728.png)

**Comment by pierluigilenoci** [comment]
@Constantin07 closed by #45
